### PR TITLE
feat: Add horizontal chart with rows of data

### PR DESCRIPTION
**Pull request by aspencer116**
This is ready for review and the only functionality that I expect to be missing is the chart being visible in IE11. We need a different solution to get that working. 

- Adds styles and react components for the horizontal chart.
- This only changes one of the horizontal charts. The pattern established here can be applied to the others.

Note, that this solution involves completely removing the table of data that was there before the chart. Extra attention should be given to accessibility during testing. I'm still new to the react things so let me know if there's any way that can be improved!

[GSS-64](https://sparkbox.atlassian.net/browse/DSS-64)

### CODE REVIEW

File: src/scss/_components.chart.horizontal.scss [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    @media (min-width: $bp-md) {
+      flex: 1 1 auto;
+      text-align: right;
+      margin: 0 1rem 0 0;
+    }
+  }
+
+  &__block {
+    flex: 1 1 auto;
+    min-width: 1.5rem; // To ensure that the element is large enough to hover
```
Comment (mdt2) [inline src/scss/_components.chart.horizontal.scss]: Nice!

File: src/scss/_utilities.scss [hunk 2]
```diff
@@ -39,3 +39,9 @@
 .util-zindex-100 {
   z-index: 100;
 }
+
+.util-delist {
```
Comment (mdt2) [inline src/scss/_utilities.scss]: There's a delist mixin that you could pull in here from `_tools.mixins.general.scss`

File: src/scss/_components.chart.horizontal.scss [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+    &--white {
+      border-color: $white;
+      background-image: linear-gradient(to bottom, rgba($white, 0.8) 50%, $black);
+      box-shadow: 0 0 0.25rem $white;
+    }
+  }
+
+  &__chart-percent {
+    // Percent value of the chart row block. Hidden until hovered
```
Comment (mdt2) [inline src/scss/_components.chart.horizontal.scss]: These comments are super helpful 💯

File: src/components/diagram-3b-table.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React from "react"
 
 const Table3b = () => (
-  <section className="cmp-diagram-table">
+  <section className="cmp-diagram">
     <p id="table3b" className="font-diagram-heading">Q: What about a client engagement leads you to suggest or not suggest a design system?</p>
     <p className="font-diagram-copy">79 Agency Responses | Select all that apply</p>
-    <div className="cmp-diagram-table__scrollable">
+    <div className="cmp-diagram__table">
```
Comment (mdt2) [inline src/components/diagram-3b-table.js]: Is the `-diagram` part confusing on these? Should the class names be something more like `cmp-table` and `cmp-chart`? Maybe we don't need `diagram` at all 🤔

File: src/scss/_utilities.scss (hunk 2 above)
Comment (aspencer116) [reply src/scss/_utilities.scss]: Nice! I'll use that instead

File: src/components/diagram-3b-table.js (hunk 4 above)
Comment (aspencer116) [reply src/components/diagram-3b-table.js]: Technically `diagram` is a container that holds a table and chart so changing it to `table` would make it redundant. This all might change with the IE11 fix too

Comment (mdt2) [reply src/components/diagram-3b-table.js]: Got it, good point 👍

**Comment by mdt2** [comment]
Merged after a local rebase.

**Pull request closed without merging**